src/main/java/com/Autopark/service/OrdersService.java

```diff
@@ -0,0 +1,29 @@
+package com.Autopark.service;
+
+import com.Autopark.entity.Orders;
+import com.Autopark.infrastructure.core.annotations.Autowired;
+import com.Autopark.infrastructure.core.annotations.InitMethod;
+import com.Autopark.infrastructure.orm.EntityManager;
+
+import java.util.List;
+
+public class OrdersService {
+    @Autowired
+    EntityManager entityManager;
+
+    @InitMethod
+    public void init() {
```

Comment:
Why are you using here empty init method?

Reply:
Because in Task say: need use empty @InitMethod in TypesService and when made OrdersService I used this model in this class too. Deleted him or what?

Reply:
Ok, it's meaning less, but leave it.
It's fine.